packages/wallet-sdk/src/relay/Session.ts

```diff
@@ -48,19 +46,6 @@ export class Session {
     return null;
   }
 
-  public static get persistedSessionIdChange$(): Observable<{
```

Comment:
what was this originally used for?

Reply:
@cb-jake i couldn't find any usages

Reply:
Sounds good, maybe @hieronymus777 has context?

Reply:
i did some digging, and it turned out this could have been deprecated along with this PR https://github.com/coinbase/coinbase-wallet-sdk/pull/319 
this is a dead code